proto/chorus/diff.proto

```diff
@@ -0,0 +1,82 @@
+syntax = "proto3";
+
+package chorus;
+
+import "google/protobuf/empty.proto";
+
+option go_package = "github.com/clyso/chorus/proto/chorus;pb";
+
+service Diff {
+  rpc StartConsistencyCheck(StartConsistencyCheckRequest)
+      returns (google.protobuf.Empty);
+
+  rpc ListConsistencyChecks(google.protobuf.Empty)
+      returns (ListConsistencyChecksResponse);
+
+  rpc GetConsistencyCheckReport(ConsistencyCheckRequest)
+      returns (GetConsistencyCheckReportResponse);
+
+  rpc GetConsistencyCheckReportEntries(GetConsistencyCheckReportEntriesRequest)
+      returns (GetConsistencyCheckReportEntriesResponse);
+
+  rpc DeleteConsistencyCheckReport(ConsistencyCheckRequest)
+      returns (google.protobuf.Empty);
+}
+
+message MigrateLocation {
+  string storage = 1;
+  string bucket = 2;
+}
+
+message StartConsistencyCheckRequest {
+  repeated MigrateLocation locations = 1;
+  string user = 2;
+  bool checkOnlyLastVersions = 3;
+  bool ignoreEtags = 4;
+  bool ignoreSizes = 5;
+}
+
+message ConsistencyCheckRequest { repeated MigrateLocation locations = 1; }
+
+message ConsistencyCheck {
+  repeated MigrateLocation locations = 2;
+  uint64 queued = 3;
+  uint64 completed = 4;
+  bool ready = 5;
+  bool consistent = 6;
+  bool versioned = 7;
+  bool withSize = 8;
+  bool withEtag = 9;
+}
+
+message ListConsistencyChecksResponse { repeated ConsistencyCheck checks = 1; }
+
+message GetConsistencyCheckReportRequest {
+  repeated MigrateLocation locations = 1;
+}
+
+message GetConsistencyCheckReportResponse { ConsistencyCheck check = 1; }
+
+message ConsistencyCheckStorageEntry {
+  string storage = 1;
+  string versionId = 2;
+}
+
+message ConsistencyCheckReportEntry {
+  string object = 1;
+  uint64 versionIdx = 2;
+  uint64 size = 3;
+  string etag = 4;
+  repeated ConsistencyCheckStorageEntry storageEntries = 5;
+}
+
+message GetConsistencyCheckReportEntriesRequest {
+  repeated MigrateLocation locations = 1;
+  uint64 Cursor = 2;
+  uint64 PageSize = 3;
```

Comment:
I assume that's something I missed, but we can rename these fields as `lowerCamelCase` to make naming uniform

Reply:
done